python/cudf/cudf/tests/test_indexing.py

```diff
@@ -2067,3 +2067,24 @@ def test_loc_index_notinindex_slice(
             expect = pdf.loc[lo:hi:take_order]
             actual = df.loc[lo:hi:take_order]
             assert_eq(expect, actual)
+
+
+@pytest.mark.parametrize(
+    "arg",
+    [
+        (2, ("one", "second")),
+        (slice(None, None, None), ("two", "first")),
+        (1, ("one", "first")),
+        (slice(None, None, None), ("two", "second")),
+    ],
+)
+def test_loc_dataframe_column_multiindex(arg):
```

Comment:
Can you add a three-level multiindex (to catch the bug in the name length above).

Reply:
Done